SofaKernel/modules/SofaMeshCollision/TriangleModel.inl

```diff
@@ -81,13 +81,13 @@ void TTriangleModel<DataTypes>::init()
 
     if (mstate==NULL)
     {
-        serr << "TriangleModel requires a Vec3 Mechanical Model" << sendl;
+        msg_error() << "TriangleModel requires a Vec3 Mechanical Model"msg_error()
```

Comment:
The msg_error() hanging around at the end of the line is what causes the build to fail on the CI, simply removing it will fix it

Reply:
i fixed it